hw03_leghlimia.py

```diff
@@ -0,0 +1,96 @@
+# Author: Mahmoud Leghlimi
+# Username: Leghlimi
+#
+# Assignment: HW03
+# Purpose:
+# Google Doc Link: https://docs.google.com/document/d/1wAef8xBkCldLnTCKJWgGhmvbp0nYVJpMk71C0vCxwEs/edit?usp=sharing
+#
+#################################################################################
+# Acknowledgements:
+#
+#
+################################################################################
+
+import turtle
+
+
+def DrawHuman(mah):
```

Comment:
Pay attention to the gray squigglies. This one indicates that functions should be lowercase (because Classes are typically defined using upper case, just as `Screen` and `Turtle`).

Reply:
Multiple oinstances of this problem.